staging/src/k8s.io/apiserver/pkg/server/filters/goaway_test.go

```diff
@@ -105,16 +105,13 @@ var (
 	// for watch request, test GOAWAY server push 1 byte in every second.
 	responseBody = []byte("hello")
 
-	// responseBodySize is the size of response body which test GOAWAY server sent for watch request,
-	// used to check if watch request was broken by GOAWAY frame.
-	responseBodySize = len(responseBody)
-
 	// requestPostBody is the request body which client must send to test GOAWAY server for POST method,
 	// otherwise, test GOAWAY server will respond 400 HTTP status code.
 	requestPostBody = responseBody
 )
 
 // newTestGOAWAYServer return a test GOAWAY server instance.
+//lint:ignore U1000 func newTestGOAWAYServer is used in func TestClientReceivedGOAWAY and func TestGOAWAYConcurrency
```

Comment:
I guess this lint error will go away after https://github.com/kubernetes/kubernetes/pull/98809 gets merged. Can you update this PR after that one is merged?

Reply:
OK, thanks.

Reply:
@caesarxuchao done, please take a look, thanks.

Reply:
#98809 might need to be reverted. https://github.com/kubernetes/kubernetes/issues/94622#issuecomment-799164426.

Let's wait for that issue solved first.

Sorry for going back and forth on this.